content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -1,88 +1,91 @@
+working on the different changes that were part of the front-end architecture
+overhaul.
 
 ## OpenTelemetry Demo Application Description and Tech Stack
 
 The demo app is an astronomy store, with basic eCommerce features such as a
-shopping cart, currency selector, payment, and checkout. It also includes
-the ability to display promotions (ads) and related products depending on
-the user’s context.
+shopping cart, currency selector, payment, and checkout. It also includes the
+ability to display promotions (ads) and related products depending on the user’s
+context.
 
 The demo’s stack includes multiple microservices in different languages,
 covering each of the following OTel-supported languages:
 
-1. Go
-2. PHP
-3. C++
-4. .NET
-5. Ruby
-6. Python
-7. Node.js
-8. Rust
-9. Elixir
+1. .NET
+2. C++
+3. Elixir
+4. Go
+5. Node.js
+6. PHP
+7. Python
+8. Ruby
+9. Rust
 
 Every microservice has a specific goal and can communicate with others by using
-a global GRPC definition. Persistent information is saved into a Postgres
+a global gRPC definition. Persistent information is saved into a PostgresSQL
 database and there are outbound services that connect with third-party services
 to trigger events (such as confirmation emails). All of the microservices,
-including the front end, are connected to the same Open Telemetry collector
-instance, which uses Jaeger as one of the data stores for the traces and spans.
+including the front-end, are connected to the same OpenTelemetry collector
+instance, which uses Jaeger as one of the data stores for the tracing data.
 
-![Open Telemetry Demo System Diagram](system-diagram.png)
+![OpenTelemetry Demo System Diagram](system-diagram.png)
 
-Prior to re-architecting, the front end consisted of a Golang Server Side Rendered app,
-which sent complete HTML to the browser for display. Every request and call
-redirected to the server so new information was shown.
+Prior to re-architecting, the front-end consisted of a Golang SSR app, which
+sent complete HTML to the browser for display. Every request and call redirected
+to the server so new information was shown.
 
 ## Web App Styling Improvements, Theme Updates, and User Experience Redesign
 
-Before starting the development process, the front end application wasn’t
+Before starting the development process, the front-end application wasn’t
 matching the theme that OpenTelemetry had been using in terms of colors,
 products, and overall user experience. In addition, the demo lacked a real
-front end (browser side) application as the current implementation was a Go
+front-end (browser side) application as the current implementation was a Go
 application.
 
-![Open Telemetry Demo Old Front end](old-design.png)
+![OpenTelemetry Demo Old Front-end](old-design.png)
 
 The first task at hand was to bring the demo to the modern age by updating the
 design, color schemes, and user experience. Olly Babiak walked into the fray to
```

Comment:
Would be good to provide a link to Olly Babiak's GitHub profile (or Twitter if GH not available).